README.md

```diff
@@ -1,4 +1,13 @@
 # IconDapp
+IconDapp is a hub of icons forall DApps on Neo3.
```

Comment:
You are using "D**A**pps" here, but on the `getContractParent` method description, it's "D**a**pps". Not sure which one is the wrong one, but it needs to be cohesive

Reply:
Fixed